monday/resources/columns.py

```diff
@@ -0,0 +1,18 @@
+from monday.resources.base import BaseResource
+from monday.query_joins import (
+    get_columns_by_board_query,
+    update_multiple_column_values_query,
+)
+
+
+class ColumnsResource(BaseResource):
+    def __init__(self, token, headers):
+        super().__init__(token, headers)
```

Comment:
Unnecessary since no change from super

Reply:
Though I notice this is likely because other Resource sub classes have an unnecessary `super().__init__` too; I think would be nice to remove